pkg/csi/service/osutils/linux_os_utils.go

```diff
@@ -1023,3 +1024,22 @@ func isTargetInMounts(ctx context.Context, target string, mnts []gofsutil.Info)
 func (osUtils *OsUtils) ShouldContinue(ctx context.Context) {
 	// no op for linux
 }
+
+// un-escapes "\nnn" sequences in /proc/self/mounts. For example, replaces "\040" with space " ".
+func unescape(in string) string {
```

Comment:
can you supply context in this function, and use create logger using

```
log := logger.GetLogger(ctx)
```

Reply:
Fixed